database/wireless_regdb.ksy

```diff
@@ -0,0 +1,189 @@
+meta:
+  id: wireless_regdb
+  title: Linux wireless regulation database (aka Central Regulatory (Domain Agent) DataBase)
+  license: copyleft-next-0.3.0 AND ISC
+  endian: be
+  encoding: ASCII
+  xref:
+    wikidata: Q11375
+
+-license-URIs:
+  - https://github.com/copyleft-next/copyleft-next/blob/master/Releases/copyleft-next-0.3.0
+
+-license-header: |
+  Copyright (c) 2013, Luis R. Rodriguez <mcgrof@do-not-panic.com>
+
+  CRDA is licensed under copyleft-next 0.3.0
+
+  Refer to copyleft-next-0.3.0 for license details, you can also always get the latest updates of copyleft-next from:
+
+  https://gitorious.org/copyleft-next/
+
+  This file incorporates work covered by the following copyright and permission notice:
+
+  Copyright (c) 2008, Luis R. Rodriguez <mcgrof@do-not-panic.com>
+  Copyright (c) 2008, Johannes Berg <johannes@sipsolutions.net>
+  Copyright (c) 2008, Michael Green <Michael.Green@Atheros.com>
+
+  Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.
+
+  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+doc: |
+  the test file: https://git.kernel.org/pub/scm/linux/kernel/git/sforshee/wireless-regdb.git/plain/regulatory.bin
+doc-ref:
+  - "https://wireless.wiki.kernel.org/en/developers/Regulatory/wireless-regdb"
+  - "https://git.kernel.org/pub/scm/linux/kernel/git/mcgrof/crda.git/tree/regdb.h"
+  - "https://git.kernel.org/pub/scm/linux/kernel/git/mcgrof/crda.git/tree/reglib.c"
+  - "https://git.kernel.org/pub/scm/linux/kernel/git/sforshee/wireless-regdb.git/tree/db2bin.py"
+seq:
+  - id: signature
+    contents: ["RGDB"]
+  - id: version
+    type: u4
+  - id: states_ptr
+    -orig-id: countries_ptr
+    type: u4
+    doc: The list is sorted alphabetically to allow binary searching (should it become really huge).
+  - id: states_count
+    -orig-id: countries_count
+    type: u4
```

Comment:
I understand the original intent behind renaming "countries" to "states", but in this case I would rather not do that:

* "states" might be confused with USA states
* "states" might be confused with different persistent states of some kind of process

Let's stick with "countries"?

Reply:
How about `jurisdictions`?

Reply:
Also `realm`s?